installer/algorand@.service

```diff
@@ -0,0 +1,37 @@
+## For some data directory $ALGORAND_DATA
+##
+## Setup:
+## ALGORAND_DATA=/var/lib/algorand_devnet
```

Comment:
Nit: Can we change the example to testnet instead of devnet?  We don't want to push people into connecting to devnet.

Reply:
swapped